tests/testdata/snapshots/deployment.yaml/disable auth using surrealdb.unauhenticated

```diff
@@ -0,0 +1,62 @@
+---
+# Source: surrealdb/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testrelease-surrealdb
+  labels:
+    helm.sh/chart: surrealdb-0.3.5
+    app.kubernetes.io/name: surrealdb
+    app.kubernetes.io/instance: testrelease
+    app.kubernetes.io/version: "1.0.0"
+    app.kubernetes.io/managed-by: Helm
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: surrealdb
+      app.kubernetes.io/instance: testrelease
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: surrealdb
+        app.kubernetes.io/instance: testrelease
+    spec:
+      serviceAccountName: testrelease-surrealdb
+      securityContext:
+        {}
+      containers:
+        - name: surrealdb
+          securityContext:
+            {}
+          image: "surrealdb/surrealdb:1.0.0"
+          imagePullPolicy: IfNotPresent
+          args:
+            - start
+          env:
+            - name: SURREAL_NO_BANNER
+              value: "true"
+            - name: SURREAL_PATH
+              value: memory
+            - name: SURREAL_LOG
+              value: info
+            - name: SURREAL_BIND
+              value: 0.0.0.0:8000
+            - name: SURREAL_AUTH
+              value: "true"
+            - name: SURREAL_UNAUTHENTICATED
+              value: "false"
```

Comment:
Similarly, this one shows that `surrealdb.unauthenticated=true` being unable to disable the auth actually.